02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,57 @@
+
+let {people} = require('../data');
+
+const getPeople = (req, res) => {
+    const newPeople = people.map((person) => {
+        const { id, name } = person
+        return { id, name }
+    })
+
+    res.status(201).json( { success: true, data: newPeople } )
+}
+
+const addPerson = (req, res) => {
+    const { name } = req.body;
+
+    if (!name) {
+        return res.status(401).json( { success: false, msg: "please provide a name!" })
+    }
+
+    people.push( { id: people.length + 1, name: name })
+    res.status(201).json( { success: true, person: name })
+}
+
+const updatePeople = (req, res) => {
+    const { name } = req.body;
```

Comment:
What happens if the name isn't provided?

Reply:
We could not update the the requested name, if the name isn't provided.